internal/wasm/memory.go

```diff
@@ -244,6 +245,10 @@ func (m *MemoryInstance) Grow(delta uint32) (result uint32, ok bool) {
 		return 0, false
 	} else if m.expBuffer != nil {
 		buffer := m.expBuffer.Reallocate(MemoryPagesToBytesNum(newPages))
+		// If the allocator fails to grow, we push -1 according to the spec.
```

Comment:
I'm probably missing something but where is the -1 here?

Reply:
same comment

Reply:
I just copied the comment: 
https://github.com/tetratelabs/wazero/blob/20de8b01b5183c29a44ee0bd13ee0cbb15210f1d/internal/wasm/memory.go#L242-L250

But that's actually what happens [one layer deep](https://developer.mozilla.org/en-US/docs/WebAssembly/Reference/Memory/Grow).

Reply:
heh could you remove the comment

Reply:
Sure. Both?

Reply:
yeah